config/runtime.exs

```diff
@@ -155,6 +155,8 @@ if otel_state != :purge,
       ]
     )
 
+config :membrane_rtc_engine_timescaledb, repo: VideoRoom.Repo
+
```

Comment:
I guess there's no point in putting it in runtime config as it cannot change - this should be compile time config as it was